hdr_schemata/models/HDRUK/2.1.2/schema.json

```diff
@@ -0,0 +1,2149 @@
+{
```

Comment:
it was indeed generated by save_schema. Will need to revisit to find a better way of doing this - i.e. generated from GitHub action rather than manually